### PR TITLE
[docs] fix the docs about exec process

### PR DESCRIPTION
**Pull request by juev**
## Description

Not all documentation has been updated previously.

## Why do we need it, and what problem does it solve?

Fixed: #8994

## What is the expected result?

The documentation contains updated information on running commands.

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: docs
type: fix
summary: Fix the docs about how to exec into the deckhnouse leader pod.
impact_level: low
```

### CODE REVIEW

File: testing/cloud_layouts/script.d/wait_cluster_ready/info_script.sh [hunk 1]
```diff
@@ -20,7 +20,7 @@ kubectl -n d8-system get deploy/deckhouse -o wide
 echo "Pod/deckhouse-*"
 kubectl -n d8-system get po -o wide | grep ^deckhouse
 echo "Enabled modules:"
-kubectl -n d8-system exec deploy/deckhouse -- deckhouse-controller module list -o yaml | grep -v enabledModules: | sort
+kubectl -n d8-system exec svc/deckhouse-leader -- deckhouse-controller module list -o yaml | grep -v enabledModules: | sort
```
Comment (juev) [inline testing/cloud_layouts/script.d/wait_cluster_ready/info_script.sh:23]: I'm not sure if changes are required here either, if necessary I'll delete it

File: testing/cloud_layouts/script.d/wait_cluster_ready/info_script_eks.sh [hunk 2]
```diff
@@ -20,7 +20,7 @@ kubectl -n d8-system get deploy/deckhouse -o wide
 echo "Pod/deckhouse-*"
 kubectl -n d8-system get po -o wide | grep ^deckhouse
 echo "Enabled modules:"
-kubectl -n d8-system exec deploy/deckhouse -- deckhouse-controller module list -o yaml | grep -v enabledModules: | sort
+kubectl -n d8-system exec svc/deckhouse-leader -- deckhouse-controller module list -o yaml | grep -v enabledModules: | sort
```
Comment (juev) [inline testing/cloud_layouts/script.d/wait_cluster_ready/info_script_eks.sh:23]: I'm not sure if changes are required here either, if necessary I'll delete it

File: testing/cloud_layouts/script.d/wait_cluster_ready/info_script.sh (hunk 1 above)
Comment (yalosev) [reply testing/cloud_layouts/script.d/wait_cluster_ready/info_script.sh:23]: Let's keep them. Doesn't look harmful

**Review by yalosev — APPROVED** [review]

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [9367](https://github.com/deckhouse/deckhouse/pull/9367) to the branch [release-1.63](https://github.com/deckhouse/deckhouse/tree/release-1.63) successful!

**Pull request merged**